sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Rename parameter `useLabelFile` to `useTrainingLabels` on `beginTraining` method in FormTrainingClients
 - Replace parameters `filePrefix` and `includeSubFolders` with `TrainingFileFilter` model
 - Rename AccountProperties `count` and `limit` to `customModelCount` and `customModelLimit`
+- Added support for AAD Authentication.
```

Comment:
Consider adding an authentication sample like Python is doing.
https://github.com/Azure/azure-sdk-for-java/pull/11156#issuecomment-628299258